package.json

```diff
@@ -1684,5 +1684,6 @@
     "xmlbuilder": "13.0.2",
     "yargs": "^15.4.1",
     "yarn-deduplicate": "^6.0.2"
-  }
-}
\ No newline at end of file
+  },
+  "packageManager": "yarn@1.22.21"
+}
```

Comment:
Sorry for all the issues! I think this is the last one - bootstrap is removing the trailing newline here and we have a modified files check that's failing.  Can we remove the newline?

Reply:
Done!